### DISCUSSION TITLE
I don't mind having the option to see recommended posts, but y'all need to think more critically about the feed.

### DISCUSSION THREAD

**Discussion by abraithwaite** [community/community#37548] [Other Feature Feedback, Questions, & Ideas]
Personally, I love the discovery aspect of github.  I suspect that recommended posts in the feed is apart of that.

However, in the short time I've been using it, I've already felt that it's much less useful than a simple following feed.

First off, there's already a means to watch releases.  I don't want starred repositories releases being shown in my feed.  That's pollution and lowers the value of my feed drastically.

Secondly, you're doing some black magic behind the scenes.  The recommendation algorithm needs a lot of work for me to want to use the feed at all.  Like, WTF is this?  I've never starred this, but it's recommended in my feed and the ant menu says I'm seeing this because I starred it?  Quit the hacks and value your customer's attention more seriously before putting garbage in front of them.

![Screen Shot 2022-10-27 at 11 22 33 AM](https://user-images.githubusercontent.com/3791413/198369379-8dde0fed-ea03-445a-a54c-095424d7d0ff.png)

I'm not calling the project suggested garbage, but do I _really_ care about another static site generator?  The answer is a firm NO.

**Comment by pawamoy** [comment]
> First off, there's already a means to watch releases. I don't want starred repositories releases being shown in my feed. That's pollution and lowers the value of my feed drastically.

Cannot agree more. Had to remove releases from the feed, and that's sad because I liked seing releases of projects I *opted* to see releases for.

**Comment by ossanna16** [comment]
Hi there @abraithwaite and welcome to our community! Thank you for the feedback 🙂
